generators/config/templates/default.yml

```diff
@@ -96,8 +96,14 @@
 ###########################################################
  mail_adapter: mailgun
  mail_wait_time: 60
+ mail_idle_timeout: 15
+ mail_sleep_time: 3
```

Comment:
Do we need this option for all email adapters? We already have mailgung_sleep_time. If so, then we need to unify the settings but with backward compatibility